web/Views/Home/_FAQPartial.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿<div class="card custom-card mt-2">
+    <div class="card-body">
+        <div class="collapse" id="faqCollapse">
+            <strong>As a tenant, what references do I need to provide to move into a property?</strong>
+            <p>References will depend on the Landlord’s criteria. These references can be emailed to the Landlord with the support of a Change Ahead volunteer.</p>

```

Comment:
I more meant that each <strong> and <p> combination could be a component, which we then pass in a single (string question, string answer). So we could just turn this into a foreach of a List<(string question, string answer)>, where we define the list in the view model

Reply:
That makes a lot more sense, I don't know how I misinterpreted that!